File: koopey.web/src/app/models/classification.ts
import { Base} from "./base/base";
import { Asset } from "../models/asset";
import { Tag } from "../models/tag";

export class Classification extends Base {
    public asset: Array<Asset> = new Array<Asset>();
    public assetId!: String;
    public tag: Array<Tag> = new Array<Tag>();
    public tagId!: String;
}

File: koopey.android/app/src/main/java/com/koopey/model/Classification.java
package com.koopey.model;

import com.koopey.model.base.Base;

import java.io.Serializable;
import java.util.UUID;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;
import lombok.experimental.SuperBuilder;
@Data
@EqualsAndHashCode(callSuper = true)
@SuperBuilder
public class Classification extends Base{

    public static final String CLASSIFICATION_FILE_NAME = "classification.dat";

    private String assetId ;
    private String tagId;
}
